src/demo.js

```diff
@@ -13,8 +13,10 @@ class Demo extends Component {
   constructor(props) {
     super(props)
     this.state = {
-      image: null,
```

Comment:
I've changed this because my console was reporting a warning about null state properties.